src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java

```diff
@@ -551,6 +551,20 @@ void setDriver(SchedulerDriver driver) {
   }
 
   @VisibleForTesting
+  String findRoleForPorts(Offer offer) {
+
+      String role = null;
```

Comment:
This might return null. I was wondering if setting 'null' for role might affect the task launch